module/VuFind/src/VuFind/DigitalContent/FakeOverdriveConnector.php

```diff
@@ -0,0 +1,325 @@
+ * @author   Demian Katz <demian.katz@villanova.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public
+ *           License
+ * @link     https://vufind.org/wiki/development Wiki
+ */
+namespace VuFind\DigitalContent;
+
+use Laminas\Config\Config;
+
+/**
+ * FakeOverdriveConnector
+ *
+ * Class responsible for simulating the Overdrive API for test purposes.
+ *
+ * @category VuFind
+ * @package  DigitalContent
+ * @author   Demian Katz <demian.katz@villanova.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public
+ *           License
+ * @link     https://vufind.org/wiki/development Wiki
+ */
+class FakeOverdriveConnector extends OverdriveConnector
+{
+    /**
+     * Constructor
+     *
+     * @param Config $mainConfig   VuFind main conf
+     * @param Config $recordConfig Record-specific conf file
+     */
+    public function __construct(
+        Config $mainConfig,
+        Config $recordConfig
+    ) {
+        $this->mainConfig = $mainConfig;
+        $this->recordConfig = $recordConfig;
+    }
+
+    /**
+     * Get (Logged-in) User
+     *
+     * Returns the currently logged in user or false if the user is not
+     *
+     * @return array|boolean  an array of user info from the ILSAuthenticator
+     *                        or false if user is not logged in.
+     */
+    public function getUser()
+    {
+        return [];
+    }
+
+    /**
+     * Get Overdrive Access
+     *
+     * Whether the patron has access to overdrive actions (hold,
+     * checkout etc.).
+     * This is stored and retrieved from the session.
+     *
+     * @param bool $refresh Force a check instead of checking cache
+     *
+     * @return object
+     */
+    public function getAccess($refresh = false)
+    {
+        return (object)[
+            'status' => true,
+        ];
+    }
+
+    /**
+     * Get Availability
+     *
+     * Retrieves the availability for a single resource from Overdrive API
+     * with information like copiesOwned, copiesAvailable, numberOfHolds et.
+     *
+     * @param string $overDriveId The Overdrive ID (reserve ID) of the eResource
+     *
+     * @return object  Standard object with availability info
+     *
+     * @link https://developer.overdrive.com/apis/library-availability-new
+     */
+    public function getAvailability($overDriveId)
+    {
+        return new \stdClass;
+    }
+
+    /**
+     * Get Availability (in) Bulk
+     *
+     * Gets availability for up to 25 titles at once.  This is used by the
+     * the ajax availability system
+     *
+     * @param array $overDriveIds The Overdrive ID (reserve IDs) of the
+     *                            eResources
+     *
+     * @return array|bool see getAvailability
+     */
+    public function getAvailabilityBulk($overDriveIds = [])
+    {
+        return [];
+    }
+
+    /**
+     * Get Colllection Token
+     *
+     * Gets the colleciton token for the Overdrive collection. The collection
```

Comment:
Typo: colleciton (ditto)

Reply:
This is still there.

Reply:
Sorry, saw the "ditto" and fixed the "Colllection" typo, then missed the different collection-related typo right below it. Should now be fully fixed. :-)